### PR TITLE
Add actions workflow to build and publish docker image

### LINKED ISSUE #1: Create an action that builds the RSD migration docker image
Create an action that builds the docker image

### PR DESCRIPTION
**Pull request by Normo**
* Closes #1

### CODE REVIEW

**Review by cmeessen — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            type=ref,event=tag
+            type=ref,event=pr
+            # set latest tag for default branch
+            type=raw,value=latest,enable={{is_default_branch}}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v3.1.0
+        with:
+          context: .
+          push: true
```
Comment (tobiashuste) [inline .github/workflows/release.yml:56]: Does this mean, that every PR is pushed to the registry as well?

**Review by tobiashuste — COMMENTED** [review]
In principle this could work, even though the workflow did not run yet.

Nevertheless, I wonder if too many images might be pushed to the registry. Not sure about that.

File: .github/workflows/release.yml (hunk 1 above)
Comment (Normo) [reply .github/workflows/release.yml:56]: Yes.

Comment (tobiashuste) [reply .github/workflows/release.yml:56]: If this is intended, I would approve it. But there is no real option to clean up the registry automatically as far as I could see.

**Review by tobiashuste — APPROVED** [review]
Thanks, @Normo!

**Pull request merged**